tide_core.drush.inc

```diff
@@ -21,7 +35,302 @@ function tide_core_drush_command() {
   ];
 
   return $items;
-}
+
+};
+
+/**
+ * Get the node id of the page to redirect to.
+ *
+ * @param string $to
+ *   URI of redirected page to get the internal path of.
+ *
+ * @return string
+ *   When an internal node id is found or if the URL is to an external site.
+ */
+function get_internal_path($to) {
+
+  if ((substr($to, 0, 4) == 'http')) {
+    return $to;
+  }
+  $to = (substr($to, 0, 1) == '/') ? $to : '/' . $to;
+  $path = \Drupal::service('path.alias_manager')->getPathByAlias($to);
+  if (preg_match('/node\/(\d+)/', $path, $matches)) {
+    return 'node/' . $matches[1];
+  }
+  return FALSE;
+};
+
+/**
+ * Uses redirect entity to save entry to DB.
+ *
+ * @param string $from
+ *   URI of page to redirect FROM.
+ * @param string $to
+ *   URI of page to redirect TO.
+ * @param string $home_url
+ *   URL of the current domain being hosted by Drupal. (e.g vic.gov.au)
+ * @param string $lang
+ *   Language code of page.
+ * @param int $status_code
+ *   Status code to return for redirect. Overridden if $to specifies code.
+ *
+ * @return bool
+ *   returns FALSE if input was invalid or a node id could not be found.
+ *
+ * @throws \Exception
+ *   If saving to the DB fails.
+ */
+function add_redirects($from, $to, $home_url, $lang, $status_code) {
+
```

Comment:
@anthony-malkoun how did this pass phpcs? Could you please check that this file is in a list of lint targets?
There should be no new line after `function`

Reply:
This has been fixed @AlexSkrypnyk